dspace-server-webapp/src/test/java/org/dspace/app/rest/PatchMetadataIT.java

```diff
@@ -0,0 +1,803 @@
+                "isAuthorOfPublication",
+                "isPublicationOfAuthor");
+
+        String adminToken = getAuthToken(admin.getEmail(), password);
+
+        // Make sure we grab the latest instance of the Item from the database before adding a regular author
+        WorkspaceItem publication = workspaceItemService.find(context, publicationItem.getID());
+        itemService.addMetadata(context, publication.getItem(),
+                "dc", "contributor", "author", Item.ANY, authorsOriginalOrder.get(0));
+        workspaceItemService.update(context, publication);
+
+        context.restoreAuthSystemState();
+
+        // Create a relationship between publication and person 1
+        MvcResult mvcResult = getClient(adminToken).perform(post("/api/core/relationships")
+                .param("relationshipType", publicationPersonRelationshipType.getID().toString())
+                .contentType(MediaType.parseMediaType
+                        (org.springframework.data.rest.webmvc.RestMediaTypes.TEXT_URI_LIST_VALUE))
+                .content("https://localhost:8080/server/api/core/items/" + publicationItem.getItem().getID() + "\n" +
+                                "https://localhost:8080/server/api/core/items/" + personItem1.getID()))
+                .andExpect(status().isCreated())
+                .andReturn();
+
+        context.turnOffAuthorisationSystem();
+
+        // Add two more regular authors
+        List<String> regularMetadata = new ArrayList<>();
+        publication = workspaceItemService.find(context, publicationItem.getID());
+        regularMetadata.add(authorsOriginalOrder.get(2));
+        regularMetadata.add(authorsOriginalOrder.get(3));
+        itemService.addMetadata(context, publication.getItem(),
+                "dc", "contributor", "author", null, regularMetadata);
+        workspaceItemService.update(context, publication);
+
+        context.restoreAuthSystemState();
+
+        // Create a relationship between publication and person 2
+        mvcResult = getClient(adminToken).perform(post("/api/core/relationships")
+                .param("relationshipType", publicationPersonRelationshipType.getID().toString())
+                .contentType(MediaType.parseMediaType
+                        (org.springframework.data.rest.webmvc.RestMediaTypes.TEXT_URI_LIST_VALUE))
+                .content("https://localhost:8080/server/api/core/items/" + publicationItem.getItem().getID() + "\n" +
+                                "https://localhost:8080/server/api/core/items/" + personItem2.getID()))
+                .andExpect(status().isCreated())
+                .andReturn();
+
+        publication = workspaceItemService.find(context, publicationItem.getID());
+        List<MetadataValue> publicationAuthorList =
+                itemService.getMetadata(publication.getItem(), "dc", "contributor", "author", Item.ANY);
+        assertThat(publicationAuthorList.size(), equalTo(5));
```

Comment:
Just a note, this sort of assertion is easier to read/understand (and shorter) if you use `assertEquals()` instead of the longer `assertThat(...., equalTo())`.  It's not a required change, but I find it slightly easier to read.

Reply:
The assertEquals is now used